interface-definitions/snmp.xml.in

```diff
@@ -123,18 +123,35 @@
           </leafNode>
           <leafNode name="oid-enable">
             <properties>
-              <help>Enable specific OIDs</help>
+              <help>Enable specific OIDs that by default are disable</help>
               <completionHelp>
-                <list>route-table</list>
+                <list>ip-route-table ip-net-to-media-table ip-net-to-physical-phys-address ip-cidr-route-table all</list>
               </completionHelp>
               <valueHelp>
-                <format>route-table</format>
-                <description>Enable routing table OIDs (ipCidrRouteTable inetCidrRouteTable)</description>
+                <format>all</format>
+                <description>Enable access to all oid</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ip-route-table</format>
+                <description>Enable oid .1.3.6.1.2.1.4.21</description>
```

Comment:
Just for visual pleasure, we can set here something like `Enable ipRouteTable / .1.3.6.1.2.1.4.21 OID`. What do you think?

Reply:
+1 for `Enable ipRouteTable xxx`